alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -872,8 +872,9 @@ private function serveLiveEntry()
 			return null;
 		}
 		
-		$this->deliveryProfile->setDynamicAttributes($this->deliveryAttributes);	
-		return $this->deliveryProfile->serve($liveStreamConfig);
+		$this->deliveryProfile->setDynamicAttributes($this->deliveryAttributes);
+		$this->deliveryProfile->setLiveStreamConfig($liveStreamConfig);	
+		return $this->deliveryProfile->serve();
```

Comment:
There are only 3 places where the liveStreamConfig is used out of the 20+ delivery types we use and passing it to the serve will require me to pass it to the buildServeFlavors function as well so I think it is better to leave it as is.